lhotse/recipes/ami.py

```diff
@@ -42,223 +42,386 @@
+                "EN2005",
+                "EN2006",
+                "EN2009",
+                "IN1001",
+                "IN1002",
+                "IN1005",
+                "IN1007",
+                "IN1008",
+                "IN1009",
+                "IN1012",
+                "IN1013",
+                "IN1014",
+                "IN1016",
+            ]
+            for meeting in MEETINGS[session]
+        ],
+        "dev": [
+            meeting
+            for session in [
+                "ES2003",
+                "ES2011",
+                "IS1008",
+                "TS3004",
+                "TS3006",
+                "IB4001",
+                "IB4002",
+                "IB4003",
+                "IB4004",
+                "IB4010",
+                "IB4011",
+            ]
+            for meeting in MEETINGS[session]
+        ],
+        "test": [
+            meeting
+            for session in ["ES2004", "ES2014", "IS1009", "TS3003", "TS3007", "EN2002"]
+            for meeting in MEETINGS[session]
+        ],
+    },
+    "full-corpus-asr": {
+        "train": [
+            meeting
+            for session in [
+                "ES2002",
+                "ES2005",
+                "ES2006",
+                "ES2007",
+                "ES2008",
+                "ES2009",
+                "ES2010",
+                "ES2012",
+                "ES2013",
+                "ES2015",
+                "ES2016",
+                "IS1000",
+                "IS1001",
+                "IS1002",
+                "IS1003",
+                "IS1004",
+                "IS1005",
+                "IS1006",
+                "IS1007",
+                "TS3005",
+                "TS3008",
+                "TS3009",
+                "TS3010",
+                "TS3011",
+                "TS3012",
+                "EN2001",
+                "EN2003",
+                "EN2004",
+                "EN2005",
+                "EN2006",
+                "EN2009",
+                "IN1001",
+                "IN1002",
+                "IN1005",
+                "IN1007",
+                "IN1008",
+                "IN1009",
+                "IN1012",
+                "IN1013",
+                "IN1014",
+                "IN1016",
+                "ES2014",
+                "TS3007",
+                "ES2003",
+                "TS3006",
+            ]
+            for meeting in MEETINGS[session]
+        ],
+        "dev": [
+            meeting
+            for session in [
+                "ES2011",
+                "IS1008",
+                "TS3004",
+                "IB4001",
+                "IB4002",
+                "IB4003",
+                "IB4004",
+                "IB4010",
+                "IB4011",
+            ]
+            for meeting in MEETINGS[session]
+        ],
+        "test": [
+            meeting
+            for session in ["ES2004", "IS1009", "TS3003", "EN2002"]
+            for meeting in MEETINGS[session]
+        ],
```

Comment:
I am not convinced that these auto-formatting changes are good, I find the older version much more readable. If you really want to reformat this file, I think there is an option in black to exclude some code lines

Reply:
Yeah I don't like how it splits the list into so many lines, but black is very opinionated. I didn't know about the excluding option --- I'll use that, thanks!